full_requirements.txt

```diff
@@ -0,0 +1,5 @@
+allennlp>=1.0,<2.7.0; python_version == '3.6'
```

Comment:
(minor comment) I would slightly rather this file to be called "requirements_full.txt" so that both files are next to each other in the DIR. The first thing people seek is "reqs" if they want to find it and seeing two reqs file make the intention immediately clear.

Reply:
This makes perfect sense, let me change the filename and we can all pretend it was always named `requirements_full.txt` 😄